cypress/objects/loginPage.js

```diff
@@ -0,0 +1,30 @@
+class LoginPage {
+    visit(){
+        cy.visit('/');
+    }
+
+    getUserValidationIndicator(){
+        return cy.get('.Select.not-valid')
+    }
+
+    clickSubmit(){
+        cy.get('[type="submit"]').click();
+    }
+
+    getSpecifiedOptionDromDropdown(option){
+        return cy.get(`[aria-label="${option}"]`)
+    }
+
+    chooseInputField(inputId){
+        cy.get(`[id="${inputId}"]`).click({force:true});
+    }
+
+    loginWithUserAndRole(user, role){
+        this.chooseInputField("loginForm.userId")
+        this.getSpecifiedOptionDromDropdown(user).click()
+        this.chooseInputField("loginForm.role")
+        this.getSpecifiedOptionDromDropdown(role).click()
+        this.clickSubmit()
+    }
+}
```

Comment:
This is a minor, but it is a common practice to have and empty line after a block of code ends